deploy/crds/api.astarte-platform.org_astartes_crd.yaml

```diff
@@ -14820,6 +14820,21 @@ spec:
                     type: string
                   image:
                     type: string
+                  maxOfflineMessages:
+                    description: The maximum number of QoS 1 or 2 messages to hold
+                      in the offline queue. Defaults to 1000. Set to -1 for no maximum
+                      (not recommended). Set to 0 if no messages should be stored
+                      offline.
+                    type: integer
+                  persistentClientExpiration:
+                    description: "This option allows persistent clients ( = clean
+                      session set to false) to be removed if they do not reconnect
+                      within 'persistent_client_expiration'. This is a non-standard
+                      option. As far as the MQTT specification is concerned, persistent
+                      clients persist forever. The expiration period should be an
+                      integer followed by one of 'd', 'w', 'm', 'y' for day, week,
+                      month, and year. \n Default: never"
```

Comment:
I think we should use something different, such as 1y, since otherwise datastream messages will pile up forever (this is likely the case for devices that will never connect again).
This clearly not the ideal solution, however we need to do this until we move outbound messages handling to data updater plant.
I think we must document this.

Reply:
I literally copypasted this from VMQ documentation/ @rbino 's plugin. Just tell me what to do here

Reply:
however if this is the rationale, the default should be == to the duration of the certificate, so 3m